### ISSUE TITLE
Odyssey Alpha install crashes EDLauncher

### ISSUE THREAD

**Issue by outsidefactor** [redmcg/wine-mono#5]
I can select the option to install the Odyssey alpha, press the Install button and it all seems to work for a few seconds, but before progressing to the Sync phase the Launcher silently closes.

**Comment by outsidefactor** [comment]
Here's some more details:

So far I have only tried this with a new Lutris install, using the Lutris-6.0 WINE runtime.

I installed a new version of ED to test this with. I installed it with the following Lutris script:

```
files:
- setup: N/A:Select setup exe for Elite Dangerous
game:
  arch: win64
  exe: drive_c/Program Files (x86)/Frontier/EDLaunch/EDLaunch.exe
  prefix: $GAMEDIR
installer:
- task:
    arch: win64
    description: Creating Wine prefix
    name: create_prefix
    prefix: $GAMEDIR
- task:
    app: arial corefonts cjkfonts vcrun2019 d3dcompiler_43 d3dcompiler_47 d3dx9
    arch: win64
    description: Installing dependencies
    name: winetricks
    prefix: $GAMEDIR
- task:
    app: ie8
    arch: win64
    description: Installing Internet Explorer 8 (necessary!))
    name: winetricks
    prefix: $GAMEDIR
- task:
    app: vcrun2010 vcrun2012 vcrun2015
    arch: win64
    description: Installing vcrun2010 vcrun2012 vcrun2015
    name: winetricks
    prefix: $GAMEDIR
- task:
    app: msls31
    description: Installing MS Line Services
    name: winetricks
    prefix: $GAMEDIR
- task:
    app: quartz
    arch: win64
    description: Installing Quartz
    name: winetricks
    prefix: $GAMEDIR
- task:
    app: arial
    arch: win64
    description: Installing Arial
    name: winetricks
    prefix: $GAMEDIR
- task:
    app: win7
    arch: win64
    description: Set Windows version to Windows 7
    name: winetricks
    prefix: $GAMEDIR
- task:
    arch: win64
    args: /silent
    description: Installing Elite Dangerous
    executable: setup
    name: wineexec
    prefix: $GAMEDIR
- task:
    name: winekill
    prefix: $GAMEDIR
system:
  env:
    DXVK_HUD: 0
    __GL_SHADER_DISK_CACHE: 1
    __GL_SHADER_DISK_CACHE_PATH: $GAMEDIR
  pulse_latency: true
wine:
  dxvk: true
```
After it installed, but before I ran it the 1st time, I used Lutris to start WINETricks for the prefix and manually removed Mono 5.1 and installed your customised install of Mono (https://github.com/madewokherd/wine-mono/releases/download/wine-mono-6.1.0/wine-mono-6.1.0-x86.msi).

At that point EDLauncher started fine. It happily downloads the normal version of the game, but the Alpha test release causes the issue I have reported here.

**Comment by outsidefactor** [comment]
OK, I have just re-installed in Steam with the Proton version as Proton-6.1-GE-2.

I skipped the ProtonDB step of installing Dot Net, and just set the prefix to Win7.

I then ran the game once to setup the Proton prefix and EDLauncher crashed as expected.

I then ran the proton fix assisted install of the custom mono 6.1.

The game launches, and not only does my HOTAS work (it didn't in the Lutris install) but it let me start installing the Odyssey Alpha, so it looks like the problem is isolated to Lutris.

Perhaps when the Lutris runtime for 6.1 comes out this will be resolved. I guess until it's confirmed as fixed we should leave this open?

**Comment by outsidefactor** [comment]
> OK, I have just re-installed in Steam with the Proton version as Proton-6.1-GE-2.
> 
> I skipped the ProtonDB step of installing Dot Net, and just set the prefix to Win7.
> 
> I then ran the game once to setup the Proton prefix and EDLauncher crashed as expected.
> 
> I then ran the proton fix assisted install of the custom mono 6.1.
> 
> The game launches, and not only does my HOTAS work (it didn't in the Lutris install) but it let me start installing the Odyssey Alpha, so it looks like the problem is isolated to Lutris.
> 
> Perhaps when the Lutris runtime for 6.1 comes out this will be resolved. I guess until it's confirmed as fixed we should leave this open?

OK, I can confirm that using the Proton-6.1-GE-2 runtime in Lutris does in fact allow Odyssey to install!

So the simple fix is to run that Runtime, which you should do anyway because it fixes HOTAS support and is more up to date anyway :)
